kernel/src/transaction/mod.rs

```diff
@@ -48,25 +77,30 @@ pub fn add_files_schema() -> &'static SchemaRef {
     &ADD_FILES_SCHEMA
 }
 
-pub(crate) static INTERMEDIATE_ADD_FILES_SCHEMA: LazyLock<SchemaRef> = LazyLock::new(|| {
-    Arc::new(StructType::new(vec![
-        StructField::not_null("path", DataType::STRING),
-        StructField::not_null(
-            "partitionValues",
-            MapType::new(DataType::STRING, DataType::STRING, true),
-        ),
-        StructField::not_null("size", DataType::LONG),
-        StructField::not_null("modificationTime", DataType::LONG),
-        StructField::not_null("dataChange", DataType::BOOLEAN),
-        StructField::nullable(
-            "stats",
-            DataType::struct_type(vec![StructField::nullable("numRecords", DataType::LONG)]),
-        ),
-    ]))
-});
+// NOTE: The following two methods are a hotfix for the fact that we do not have a proper SchemaBuilder yet.
```

Comment:
nit:
```suggestion
// NOTE: The following two methods are a workaround for the fact that we do not have a proper SchemaBuilder yet.
```

hotfix has a different meaning, typically an urgent fix that doesn't follow standard release process